lib/Cake/View/Errors/fatal_error.ctp

```diff
@@ -36,4 +36,3 @@
 if (extension_loaded('xdebug')) {
 	xdebug_print_function_stack();
 }
-?>
```

Comment:
Well, I think we did. Because mainly for ctp PHP blocks it makes it impossible to move them around in the template if you omit the closing tags.